surrogates/gpt.js

```diff
@@ -71,6 +71,7 @@
     };
     const gptObj = {
         _loadStarted_: true,
+        slotmap: [],
```

Comment:
this is an object on https://www.dwell.com/, why are we going with an array?
![obj](https://user-images.githubusercontent.com/985504/140829278-51a9c716-5fca-44f6-b6e4-c6e5df95a176.jpg)